### ISSUE TITLE
[CS2] Failing test rejecting inconsistent whitespace in .litcoffee files

### ISSUE THREAD

**Issue by GeoffreyBooth** [jashkenas/coffeescript#4336]
@eelco Since #4313 was merged into `2`, there is one failing test:

```
failed 1 and passed 724 tests in 2.05 seconds

  test/literate.litcoffee:56:5: error: indentation mismatch
				test "tabbed code", ->
				^
```

If I delete everything above this test in `literate.litcoffee`, leaving just:

```litcoffee
Tabs work too:

				test "tabbed code", ->
					ok yes
```
Then this test passes. See [comment](https://github.com/jashkenas/coffeescript/pull/4313#issuecomment-250924753).

**Comment by GeoffreyBooth** [comment]
Could #3924 help with this? That seems like a worthy PR in its own right.

**Comment by GeoffreyBooth** [comment]
@jashkenas @lydell what do you want to do about this? It’s causing issues with other `2`-branch pull requests to have this broken test in the `2` branch. Should we comment it out or delete it until this issue is patched? Split the test into a separate `.litcoffee` file that only uses tabs (which doesn’t fix this issue, but rather simply makes the test pass)? Or roll back #4313 until this is fixed?

**Comment by GeoffreyBooth** [comment]
Update: potential fix in #4345.
